src/index.html

```diff
@@ -6,7 +6,440 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="page__header">
+      <a href="#home" class="page__headerLogo">
+        <img
+          src="images/logo.png"
+          alt="Logo imagine"
+          class="img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="Apple" class="nav_links is-active">Apple</a>
```

Comment:
do not make it plural for no reason
```suggestion
            <a href="Apple" class="nav_link is-active">Apple</a>
```